packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html

```diff
@@ -1,5 +1,6 @@
 <div class="container-fluid mx-0 px-0 manager-on-boarding-layout">
-    <div class="row align-items-center">
+    <div data-ng-if="!$ctrl.hideIllustrationSection"
+         class="row align-items-center">
         <div class="w-100">
             <div class="placeholder p-5 d-flex justify-content-center">
```

Comment:
Is this a new demand ? We just refactored all onboarding pages for the PNR project that is still in beta, i don't think this complies with the new onboarding pages format.

Reply:
Yes, i know, but all the empty pages are now supposed to have the same pattern, if no image, then there is a default image displayed, it's not hidden. 

I think you should talk with Mounir before doing this, as everything have been harmonized

Reply:
Thanks for info.
I will discuss with him about this point 👍.